doc/getting-started/getting-started.rst

```diff
@@ -637,9 +637,9 @@ In the following steps, you will configure GRUB on the target system.
            insmod gzio
            insmod part_gpt
            insmod ext2
-           search --no-floppy --fs-uuid --set <UUID>
+           search --no-floppy --fs-uuid --set "UUID"
            echo 'loading ACRN...'
-           multiboot2 /boot/acrn/acrn.bin  root=PARTUUID=<PARTUUID>
+           multiboot2 /boot/acrn/acrn.bin  root=PARTUUID="PARTUUID"
            module2 /boot/vmlinuz-5.10.52-acrn-sos Linux_bzImage
          }
 
```

Comment:
Please add an example of the above text with the UUID and PARTUUID filled in correctly. This will give our customers confidence that they are doing the step correctly. (In the DX study, we saw that participants were not sure whether or not to include quotation marks. Some participants suggested adding an example for clarification.)